src/Umbraco.Web/PropertyEditors/MultiUrlPickerPropertyEditor.cs

```diff
@@ -13,7 +13,7 @@ namespace Umbraco.Web.PropertyEditors
     [DataEditor(
         Constants.PropertyEditors.Aliases.MultiUrlPicker,
         EditorType.PropertyValue,
-        "Multi Url Picker",
+        "Multi URL Picker",
```

Comment:
I don't think changing the data editor name should give any issues, as it's only used for displaying in the back-office, but maybe this can be double checked?